lib/providers/measures_provider.dart

```diff
@@ -0,0 +1,126 @@
+///    This program is free software: you can redistribute it and/or modify
+///    it under the terms of the GNU General Public License as published by
+///    the Free Software Foundation, either version 3 of the License, or
+///    (at your option) any later version.
+///
+///    This program is distributed in the hope that it will be useful,
+///    but WITHOUT ANY WARRANTY; without even the implied warranty of
+///    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+///    GNU General Public License for more details.
+///
+///    You should have received a copy of the GNU General Public License
+///    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import 'package:covid19mobile/model/measures_model.dart';
+import 'package:flutter/material.dart';
+
+/// Creates a [MeasuresModel] Provider for updating UI for
+class MeasuresProvider with ChangeNotifier {
```

Comment:
I think here is not needed to have all the fields because it will be a list,

so my idea is:
Api request measures, then parse the json to a list<MesureModel> and then on provider:

In UI will call the get messures into the list, 
```suggestion
class MeasuresProvider with ChangeNotifier {

List<MesureModel> _mesures = List<MesureModel>();
List<MesureModel> get mesures => _mesures;

void setMesures(List<MesureModel> values) {
_mesures = values;
notifyListeners();
}
}
```

then to go to details page pass one MesureModel to Deails page